src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -359,14 +358,19 @@ public function initialized($id)
     public function getServiceIds()
     {
         $ids = array();
+        $reversedMethodMap = array_change_key_case(array_flip($this->methodMap), \CASE_LOWER);
```

Comment:
Tests proves it works... you cant imagine how creative users can be. Sorry, couldnt resist ;-)

> $this->methodMap is a compilation artifact
What if we make it feature?